# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/media-conduit/moz.build
# Commit: ffdbeba93fa6
# Full Hash: ffdbeba93fa632a830138cb79c53cfc15bf1f13d
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 1
# Description:
#   Bug 1376873 - WebrtcMediaDataDecoderCodec updates; r=pehrsons
#   
#   This makes the ImageBuffer class implement VideoFrameBuffer class and makes
#   it available in MediaPipeline. To avoid exposing MediaPipeline to a lot of
#   internal details of WebrtcMediaDataDecoder, ImageBuffer is moved to its
# ==============================================================================

diff -r d12ad321047d -r ffdbeba93fa6 media/webrtc/signaling/src/media-conduit/moz.build
--- a/media/webrtc/signaling/src/media-conduit/moz.build	Tue Aug 21 13:40:10 2018 -0400
+++ b/media/webrtc/signaling/src/media-conduit/moz.build	Tue Aug 21 13:40:45 2018 -0400
@@ -14,6 +14,7 @@
     '/media/webrtc/signaling/src/common',
     '/media/webrtc/signaling/src/common/browser_logging',
     '/media/webrtc/signaling/src/common/time_profiling',
+    '/media/webrtc/signaling/src/media-conduit',
     '/media/webrtc/signaling/src/peerconnection',
     '/media/webrtc/trunk',
     '/media/webrtc/trunk/webrtc',